hal/halx86/generic/sysinfo.c

```diff
@@ -12,8 +12,50 @@
 #define NDEBUG
 #include <debug.h>
 
+NTSTATUS
+NTAPI
+HaliHandlePCIConfigSpaceAccess(BOOLEAN IsRead,
+                               ULONG Port,
+                               ULONG Length,
+                               PVOID Buffer);
```

Comment:
No, there's no any: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=HaliHandlePCIConfigSpaceAccess.
Maybe move it in some header file then? :thinking:

Reply:
Yes, that's what I meant in this case.

> Maybe move it in some header file then?

Reply:
You don't need this declaration at all.

Reply:
Oh, OK. I'll delete this. I wonder why Vadim added it at all.

Reply:
Oops. :sweat_smile: Looks like this declaration is badly required, because it is used in the structure below, for `&HaliHandlePCIConfigSpaceAccess` member. After removing this, I got the following compilation error: `../hal/halx86/generic/sysinfo.c:35:25: error: 'HaliHandlePCIConfigSpaceAccess' undeclared here (not in a function); did you mean 'HalHandlerForConfigSpace'?
     { 0x0CF8, 0x08, 1, &HaliHandlePCIConfigSpaceAccess }, // PCI configuration space
                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                         HalHandlerForConfigSpace`

Reply:
So I'll keep it as-is.

Reply:
Fine with me. You could also move it to before the `HaliQuerySystemInformation` declaration in halp.h.

Reply:
Fixed, thanks.